responsibleai/responsibleai/rai_insights/rai_insights.py

```diff
@@ -471,6 +473,16 @@ def _validate_rai_insights_input_parameters(
                     columns=[target_column], axis=1)
                 small_test_data = small_test_data.drop(
                     columns=[target_column], axis=1)
+                if len(small_train_data.columns) == 0 or \
+                        len(small_test_data.columns) == 0:
+                    if has_dropped_features:
+                        raise UserConfigValidationException(
+                            "All features have been dropped from the dataset"
+                        )
+                    else:
+                        raise UserConfigValidationException(
+                            "There is no feature in the dataset"
+                        )
```

Comment:
This should be actionable telling user what they need to do to avoid running into the exception again. Maybe add another blurb saying that "Please make sure that you dataset has at least one feature."?

Reply:
updated